libs/cardano-ledger-test/cardano-ledger-test.cabal

```
@@ -152,7 +162,7 @@ test-suite cardano-ledger-test
     ghc-options:
         -Wall -Wcompat -Wincomplete-record-updates
         -Wincomplete-uni-patterns -Wpartial-fields -Wredundant-constraints
-        -Wunused-packages -rtsopts -threaded
+        -rtsopts -threaded
```

That is an important warning. Pleas don't disable compiler warnings without a really good reason. 
```suggestion
        -Wunused-packages -rtsopts -threaded
```

---

Yes, this was a temporary work-around to make it possible to start a repl. For some reason cabal repl gives a bunch of spurious unused-package warnings.

---

@UlfNorell You can add this to you `cabal.project.local`, which will turn the errors into warnings only for you locally
```
if impl(ghc < 9.8)
  program-options
    ghc-options: -Wwarn
```

Note, that this is necessary `if impl(ghc < 9.8)` due to some cabal peculiarity, since that is how errors are turned on in `cabal.project`